### PR TITLE
[15.0][MIG] mgmtsystem_nonconformity_mrp: Migration to 15.0

### PR DESCRIPTION
**Pull request by mymage**

### CODE REVIEW

File: .github/workflows/pre-commit.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v4
```
Comment (primes2h) [inline .github/workflows/pre-commit.yml]: This should be reverted.
Already merged here https://github.com/OCA/management-system/commit/0e46bd9add9f5c90fa1b92b3146ff1ea501afb1d

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -113,7 +113,7 @@ repos:
           - requirements.txt
           - --header
           - "# generated from manifests external_dependencies"
-  - repo: https://gitlab.com/PyCQA/flake8
+  - repo: https://github.com/PyCQA/flake8
```
Comment (primes2h) [inline .pre-commit-config.yaml]: Same here.

**Review by primes2h — CHANGES_REQUESTED** [review]

**Review by max3903 — APPROVED** [review]

**Pull request merged**